src/UIContainer.ts

```diff
@@ -470,6 +492,18 @@ export class UIContainer extends HTMLElement {
                     receiver.fullscreen = hasValue;
                 }
             }
+        } else if (attrName === Attribute.DEVICE_TYPE) {
+            toggleAttribute(this, Attribute.MOBILE, newValue === 'mobile');
```

Comment:
Nit-pick : Maybe create from 'mobile' and 'tv' a const

Reply:
Disagree. For the public API, we prefer string unions over enums.